core/templates/components/certificate-assessment-offering-helper/certificate-offering-review-and-availability.component.html

```
@@ -0,0 +1,12 @@
+<div class="oppia-certificate-offering-review-and-availability">
+  <h2>Review & Availability</h2>
+  <!--
+    Real certificate summary, readiness breakdown, and validation errors are
+    added after the details and topic-selection data contracts are finalized.
+  -->
+
+  <div class="oppia-certificate-offering-review-actions">
+    <button (click)="onBackClicked()">Back</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)